wordpress-mobile/WordPress-iOS
# Zendesk Mobile: Adding new controller to display Support view

Fixes #9044 

**NOTES:**
- This recreates the Support table previously created by `SupportViewController`.
- Right now, it can be accessed by the 'Help' button on the nav bar. Replacing all instances of `SupportViewController` will come later.
- This is just the UI. Table row actions will come later.

**To test:**
- With `zendeskMobile` feature flag enabled:
  - Press 'Help' on the nav bar (ex: from Site Creation or Log In).
  - Verify the new view appears.
  - The new view has an additional 'My Tickets' option.
  - Selecting a row will present a temporary alert.
- With `zendeskMobile` feature flag disabled:
  - Press 'Help' on the nav bar.
  - Verify the old view appears.

**New View:**
![new_view](https://user-images.githubusercontent.com/1816888/38343109-5d8af0f2-383f-11e8-91a2-4a4acbb09f6d.png)

**Old View:**
![old_view](https://user-images.githubusercontent.com/1816888/38343123-6c875a5a-383f-11e8-8d40-8d8d4b9ef133.png)

Temporary Alert:
<img width="250" alt="temp_alert" src="https://user-images.githubusercontent.com/1816888/38343155-8a36f8ee-383f-11e8-9fff-b9381c5b2b09.png">
